telegram/bot.py

```diff
@@ -222,6 +207,23 @@ def __init__(
                 private_key, password=private_key_password, backend=default_backend()
             )
 
+    # TODO: After https://youtrack.jetbrains.com/issue/PY-50952 is fixed, we can revisit this and
+    # consider adding Paramspec from typing_extensions to properly fix this. Currently a workaround
+    def log(  # type: ignore[no-untyped-def] # skipcq: PY-D0003
+        func: Any, *args, **kwargs  # pylint: disable=W0613
```

Comment:
I think we can actually drop `*args, **kwargs`, right? I don't recall why they are here, but IISC we would only need them if we did things like `@log(*args)`
and we wanted to type out pylint error descriptions …

Reply:
yep, even I didn't know why they are here when we don't use them